bitnami/cassandra/templates/configuration-cm.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if and (.Files.Glob "files/conf/*") (not .Values.existingConfiguration) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "cassandra.fullname" . }}-configuration
```

Comment:
Shouldn't you use `{{ template "cassandra.configurationCM" . }}`?

Reply:
No, that's for obtaining whether to use the one we generate or the one provided in values.yaml